Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -1025,6 +1025,13 @@ PatchForceInternalDiskIcons (
     return EFI_NOT_FOUND;
   }
 
+  //
+  // Override patch count to 2 on macOS 13.3+ (Darwin 22.4.0).
+  //
+  if (OcMatchDarwinVersion (KernelVersion, KERNEL_VERSION (KERNEL_VERSION_VENTURA, 4, 0), 0)) {
+    mIOAHCIPortPatch.Count = 2;
```

Comment:
Set to 1 on else, in case booting an older macOS version fails we may end up patching twice on older systems.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/272802332db8030dfbb047d15016d97c751abd69